### ISSUE TITLE
Toggle Toolbar throws an error

### ISSUE THREAD

**Issue by rbenson** [MattFlower/organized#2]
**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Microsoft Windows 10 Pro
**Thrown From**: [organized](https://github.com/MattFlower/organized) package, v0.4.2
### Stack Trace

Uncaught TypeError: Cannot read property 'toggleToolbar' of undefined

```
At /C:/Users/Ryan/.atom/packages/organized/lib/toolbar.coffee:29

TypeError: Cannot read property 'toggleToolbar' of undefined
    at atom-text-editor.subscriptions.add.atom.commands.add.organized:toggleToolbar (file:///C:/Users/Ryan/.atom/packages/organized/lib/toolbar.coffee:25:118)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\Ryan\AppData\Local\atom\app-1.10.2\resources\app.asar\src\command-registry.js:260:29)
    at CommandRegistry.handleCommandEvent (C:\Users\Ryan\AppData\Local\atom\app-1.10.2\resources\app.asar\src\command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\Ryan\AppData\Local\atom\app-1.10.2\resources\app.asar\src\command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (C:\Users\Ryan\AppData\Local\atom\app-1.10.2\resources\app.asar\src\atom-environment.js:1109:28)
    at EventEmitter.outerCallback (C:\Users\Ryan\AppData\Local\atom\app-1.10.2\resources\app.asar\src\application-delegate.js:329:25)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
```

```
```

**Comment by MattFlower** [comment]
Hey @rbenson, sorry for the trouble.  It's fixed in 0.5.0.

-m

**Comment by rbenson** [comment]
I tried to update to 0.5.0 and it failed

**Comment by MattFlower** [comment]
Did you get the same error or was it something different?

**Comment by rbenson** [comment]
The whole package failed. I had to try three times

**Comment by MattFlower** [comment]
Working on it -- I haven't had a lot of luck reproducing this OS X, so I'm putting together a Windows virtual machine to make sure that isn't the problem.  In the meantime, if anyone else happens across this bug and is also seeing the same thing on a mac, please let me know.

**Comment by MattFlower** [comment]
Seemed to work on windows 10, so it wasn't that.  

I did notice something incorrect, so I decided to fix it in hopes that it might solve the trouble.  The toolbar functionality seems to pop up a dialog box when installing that asks if you want to install the tool-bar package.  Not sure why that might be a problem, but it's the only thing unusual I'm seeing during install so it seems worth a try.

Let me know if that changes anything.  If you are able to get a new stack trace that would definitely help.  I can't seem to reproduce at this point.
